build.gradle

```diff
@@ -45,6 +45,9 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-webflux")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
     implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
```

Comment:
Edit the line to follow the convention of the rest of the file, i.e., parenthesis and double quotes for string.

Reply:
Changes made to follow the convention